packages/server/src/findGraphQLTags.js

```diff
@@ -63,7 +63,11 @@ export function findGraphQLTags(
         fragments.properties.forEach(property => {
           const tagName = getGraphQLTagName(property.value.tag);
           const template = getGraphQLText(property.value.quasi);
-          if (tagName === 'graphql' || tagName === 'graphql.experimental') {
+          if (
+            tagName === 'graphql' ||
+            tagName === 'gql' ||
+            tagName === 'graphql.experimental'
+          ) {
```

Comment:
ah! so it's since been refactored under the hood, and now we handle this somewhere else: 
https://github.com/graphql/graphiql/blob/e4cec0ad7f65db2709c54dbb3a68fe49031de40b/packages/graphql-language-service-server/src/MessageProcessor.js#L689

thank you though!